Jenkinsfile

```diff
@@ -1,17 +1,33 @@
 #!/usr/bin/env groovy
 
 pipeline {
-    agent { label 'ruby' }
+    // All Linux agents have ruby + bundle installed through `asdf`
+    agent { label 'linux-amd64-docker' }
 
     options {
         buildDiscarder(logRotator(numToKeepStr: '10'))
         timeout(time: 15, unit: 'MINUTES')
         timestamps()
     }
 
+    environment {
+        // To allow using ASDF shims
+        PATH = "${env.PATH}:/home/jenkins/.asdf/shims:/home/jenkins/.asdf/bin"
```

Comment:
why not add this to the path of the image?

Reply:
I wasn't able to have a custom PATH working in the VMs. I assume it might be related to the current SSH configuration (infra.ci uses EC2 VM agents so SSH agents) that filters out.

None of the usual `profile` and `bashrc` are not loaded.